### DISCUSSION TITLE
No reply even the ETA is over

### DISCUSSION THREAD

**Discussion by DaoaLeo** [community/community#20945] [GitHub Education] [answered]
Hey, so i submitted my request on 16th july 2022, and github give me an eta of 3 days(around 19 july),but its now 20th july and i dont get any responses yet, any idea what happened ?

**Comment by MrSarno** [comment, marked as answer]
The 'E' stands for 'estimated'; it's possible there was either a surge of requests or some staff were unexpectedly unavailable.

You'll probably hear back either today or tomorrow - if it's only one day longer than originally expected, I wouldn't worry about it.
